pandas/core/arrays/datetimes.py

```diff
@@ -857,8 +857,8 @@ def tz_localize(self, tz, ambiguous="raise", nonexistent="raise") -> DatetimeArr
         This method takes a time zone (tz) naive Datetime Array/Index object
         and makes this time zone aware. It does not move the time to another
         time zone.
-        Time zone localization helps to switch from time zone aware to time
-        zone unaware objects.
+        Time zone localization helps to switch from time zone unaware to time
+        zone aware objects.
```

Comment:
the tz is None the existing is correct - why don't we break this down into multiple statements

Reply:
Hey @jreback let's clarify.

I was considering to remove the entire sentence, because the above two sentence already say what's happening:

> Localize tz-naive Datetime Array/Index to tz-aware Datetime Array/Index.

> This method takes a time zone (tz) naive Datetime Array/Index object  and makes this time zone aware.

This third sentence says the opposite:

> Time zone localization helps to switch from time zone aware to time zone unaware objects.

The first two sentences are correct, the third one is not.

This assumes that "tz-naive" means the same as "tz-unaware" or just "unaware" which I think we agree on, do we? :)

Maybe I miss something, would love to understand what you mean with "tz is None".

Reply:
I think this sentence tries to (poorly) describe that `tz_localize(None)` can convert tz-aware to tz-unaware (there's an example in this doc-string already)

So it would be great to have this in a new paragraph clarifying this point.

Reply:
Oooooh. Gotcha. And now I understand that "the tz is None the existing is correct" was supposed to be 

> *if* `tz` is `None`, ...

:pray: 

I'll happily improve on the current patch. Thanks everyone.